bin/_util.js

```diff
@@ -2,9 +2,6 @@
 
 // NOTE: Do not use directly. Extend this to support re-writing eds-import-from-figma for non-enterprise use cases
 class AbstractFigmaReader {
-  static TIER_1_MODE = '6181:0';
```

Comment:
Main goal of this change is to remove these very specific and brittle values for the script. instead of hard-coding them, we pull data from the figma payload which map to the dynamic values stored in figma itself. Guidance provided on which ones mean what.